msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards are known for their ability to adapt to the host application's overall styling. However, their adaptability is limited when it comes to the layout. The static layout approach results in unsatisfactory user experiences, especially across different screen sizes and resolutions.
+
+Adaptive Card responsive layout allows you to design Adaptive Cards that adapt to different screen sizes in Teams. Responsive Adaptive Cards dynamically adjust their layout based on the available screen width, ensuring optimal display regardless of the device or screen size. When you design an Adaptive Card for your app, use the `targetWidth` property to make your responsive and have a better user experience on Teams mobile or desktop, regardless of the screen size.
+
+#### Configure Adaptive Card responsive layout
+
+The following are the key elements to create a responsive Adaptive Card:
+
+* `CardElement.targetWidth`: Allows you to control the visibility of any element by linking it to `hostWidth`. Add the `targetWidth` property for all elements in an Adaptive Card to define different layouts for different card widths. The following are the predefined values for the `targetWidth` property:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `very narrow` | The `targetWidth` for right narrow pane in the meeting chat. |
+  | `narrow` | The `targetWidth` for mobile screen. |
+  | `standard` | The `targetWidth` for chats and channels in desktop. |
+  | `wide` | The `targetWidth` for any screen size beyond `standard`. |
+
+  The values for `targetWidth` refers to the relative width of the card and allows more flexibility in adapting to different screen sizes.
+
+  > [!NOTE]
+  > We recommend to use the predefined values for the `targetWidth` property for your Adaptive Card and not the pixel sizes of screen.
+
+* `CardElement.hostWidth`: Allows the host application to define the width it provides to the card through `targetWidth` property. The host application is responsible for mapping the actual widths to these named values. The following are the predefined values for the `hostWidth` property:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `very narrow` | The `hostWidth` for right narrow pane in the meeting chat. |
+  | `narrow` | The `hostWidth` for mobile screen. |
+  | `standard` | The `hostWidth` for chats and channels in desktop. |
+  | `wide` | The default value for `hostWidth`. |
+
+  You can add the `targetWidth` property to any element with a prefix to the predefined value that is `atLeast` or `atMost`. When you specify the targetWidth value to a predefined value with a prefix, the elements are displayed or hidden based on the `hostWidth`, providing you with the control over layout responsiveness.
```

Comment:
Please do not refer to "predefined values" in this doc. As I mentioned before, `veryNarrow`, `narrow`, `standard` and `wide` are not "predefined values" they are the only valid values.